### PR TITLE
04/api request

### PR DESCRIPTION
**Pull request by Leandro97**

### CODE REVIEW

File: stant-album/Album.swift [hunk 1]
```diff
@@ -8,16 +8,27 @@
...
+        case albumName   = "album_name"
+        case albumCover  = "album_cover"
+    }
+    
+    init(id: String,
+         artistName:  String,
          recordLabel: String,
          albumName:   String,
          albumCover:  String) {
+        self.id = id
```
Comment (liuvasconcelos) [inline stant-album/Album.swift]: Identation

File: stant-album/Request.swift [hunk 2]
```diff
@@ -0,0 +1,24 @@
+//
+//  Request.swift
```
Comment (liuvasconcelos) [inline stant-album/Request.swift]: What are you requesting? hehe.  Always put legible names on classes.  Is it a "util" class?

File: stant-album/Request.swift [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+//  Created by Leandro Martins on 02/10/19.
+//  Copyright © 2019 Stant. All rights reserved.
+//
+
+import Foundation
+
+import Foundation
+let session = URLSession(configuration: .default)
+
+//Error tratement needed
```
Comment (liuvasconcelos) [inline stant-album/Request.swift]: Remove comment .

**Review by liuvasconcelos — CHANGES_REQUESTED** [review]

File: stant-album/Album.swift (hunk 1 above)
Comment (Leandro97) [reply stant-album/Album.swift]: Done.

File: stant-album/Request.swift (hunk 3 above)
Comment (Leandro97) [reply stant-album/Request.swift]: Done.

File: stant-album/Request.swift (hunk 2 above)
Comment (Leandro97) [reply stant-album/Request.swift]: Done.

**Pull request merged**